### PR TITLE
{Include and Source} Graphical User Interface Demo

### PR DESCRIPTION
**Pull request by Tobiabass11**
I have committed the header and cpp files for the GUI to the repository. I will add the other files after the necessary feedback on how to go about it.

### CODE REVIEW

**Review by avinashkrishnan2020 — COMMENTED** [review]
@Tobiabass11 - remove .DS_Store file from include/guiheaderfiles

File: include/guiheaderfiles/customerdialog.h [hunk 1]
```diff
@@ -0,0 +1,44 @@
+#ifndef CUSTOMERDIALOG_H
+#define CUSTOMERDIALOG_H
+
+#include <QDialog>
+#include <QTimer>
+#include <string>
+#include <vector>
+using namespace std;
```
Comment (avinashkrishnan2020) [inline include/guiheaderfiles/customerdialog.h]: adding "using namespace std" isn't recommended. Remove this and add std::string, etc. (similar to UI::customerDialog in line 40)

File: src/customerdialog.cpp [hunk 2]
```diff
@@ -0,0 +1,259 @@
...
+
+//DECLARE VARIOUS VECTORS
+vector<stock> v;
+//vector<stock> vv;
+double totalPrice = 0.00; //for total price
+vector<stock> itemsArray;
+
+
+/*creating the inventory using array
+string productSerialNumber[5] = {"S4PYX", "9BJ2D","4QDI0", "7FGLO", "CBHM3"};
```
Comment (avinashkrishnan2020) [inline src/customerdialog.cpp]: remove unwanted codes that have been commented out

File: src/mainwindow.cpp [hunk 3]
```diff
@@ -0,0 +1,37 @@
+#include "mainwindow.h"
+#include "ui_mainwindow.h"
+#include <QMessageBox>
+#include <QPixmap>
+
+MainWindow::MainWindow(QWidget *parent)
```
Comment (avinashkrishnan2020) [inline src/mainwindow.cpp]: Remove unwanted code that have been commented out.

**Comment by avinashkrishnan2020** [comment]
@Tobiabass11 - checkout the comments.

**Comment by avinashkrishnan2020** [comment]
@Tobiabass11 - checkout new comments

**Comment by Tobiabass11** [comment]
I have created a new c++ class with header and source files in their appropriate folders

File: src/customerdialog.cpp (hunk 2 above)
Comment (Tobiabass11) [reply src/customerdialog.cpp]: I have removed these files and included the new GUI header and source files

**Pull request merged**